.devcontainer/devcontainer.json

```diff
@@ -4,35 +4,28 @@
 	"name": "Node.js & TypeScript",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
 	"image": "mcr.microsoft.com/devcontainers/typescript-node:1-18",
-
 	// Features to add to the dev container. More info: https://containers.dev/features.
 	"features": {
 		"ghcr.io/devcontainers/features/node:1": {}
 	},
-
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
-
 	// Use 'postCreateCommand' to run commands after the container is created.
 	"postCreateCommand": ". ${NVM_DIR}/nvm.sh && nvm install --lts",
-
-	"postStartCommand": "yarn install",
-
+	"postStartCommand": "npm install",
 	// Configure tool-specific properties.
 	"customizations": {
-	// Configure properties specific to VS Code.
-	"vscode": {
-		// Set *default* container specific settings.json values on container create.
-		"settings": { 
-		},
-		
-		// Add the IDs of extensions you want installed when the container is created.
-		"extensions": [
-			"esbenp.prettier-vscode",
-			"ms-vscode.makefile-tools"
-		]
-	}
-},
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {},
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"esbenp.prettier-vscode",
+				"ms-vscode.makefile-tools"
+			]
+		}
+	},
 	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
 	// "remoteUser": "root"
-}
+}
```

Comment:
Missing newline at the end of file. I think this changes can be reverted.

Reply:
```suggestion
}

```